### ISSUE TITLE
Screenshots for F-Droid

### ISSUE THREAD

**Issue by mondlicht-und-sterne** [mupen64plus-ae/mupen64plus-ae#885]
You can add Screenshots for F-Droid, too, if you want.

https://f-droid.org/en/docs/All_About_Descriptions_Graphics_and_Screenshots/

https://gitlab.com/snippets/1895688

**Comment by fzurita** [comment]
Yeah, I know, I was kind of lazy and didn't do that for my last f-droid update. I will take a pull request though, otherwise it will probably wait until I do the next f-droid update.

**Comment by mondlicht-und-sterne** [comment]
You only need to enter the screenshots into your fastlane. 
if you mean me by pull request, I can't.

**Comment by fzurita** [comment]
Does fastlane automatically pull from latest master branch?
